File: Service_eHospital_DongNai_A_NSTL/Implement/NS_BANGCAP_UVService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_BANGCAP_UVService : BaseService<NS_BANGCAP_UV, INS_BANGCAP_UVRepository>
    , INS_BANGCAP_UVService
    {
    private readonly INS_BANGCAP_UVRepository _NS_BANGCAP_UVRepository;
    public NS_BANGCAP_UVService(INS_BANGCAP_UVRepository NS_BANGCAP_UVRepository) : base(NS_BANGCAP_UVRepository)
    {
    _NS_BANGCAP_UVRepository = NS_BANGCAP_UVRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_BANGCAP_UV.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_BANGCAP_UV } from './NS_BANGCAP_UV.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_BANGCAP_UVService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Id', 'MaUngVien', 'TenBangcap', 'ChuyenNganh', 'Noicapbang', 'Ngaycapbang', 'Xeploai'];

    List: NS_BANGCAP_UV[] | undefined;
    ListFilter: NS_BANGCAP_UV[] | undefined;
    FormData!: NS_BANGCAP_UV;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_BANGCAP_UV";
    }
}

